app/models/user.rb

```diff
@@ -29,6 +29,11 @@ def sucard_number=(card_number)
     self.library_id = card_number[/\d{5}(\d+)/, 1]
   end
 
+  # Prefer the patron barcode from the ILS, but fall back to the library ID as-provided
```

Comment:
Can we add to the comment to say "so that the system can function when the ILS is offline"?